src/argilla/server/telemetry.py

```diff
@@ -91,16 +99,29 @@ async def track_bulk(task: TaskType, records: int):
     _CLIENT.track_data(action="LogRecordsRequested", data={"task": task, "records": records})
 
 
-async def track_login(request: Request, username: str):
+async def track_login(request: Request, user: User):
     _CLIENT.track_data(
         action="UserInfoRequested",
         data={
-            "is_default_user": username == "argilla",
-            "user_hash": str(uuid.uuid5(namespace=_CLIENT.server_id, name=username)),
+            "is_default_user": user.username == "argilla",
```

Comment:
Maybe we can use here the constant from `argilla/server/constants.py`?

```suggestion
            "is_default_user": user.username == DEFAULT_USERNAME,
```